common/cb_types.txt

```diff
@@ -2321,7 +2321,7 @@ establish_protectorate_casus_belli = {
 	
 	badboy_factor = 15
 	prestige_factor = 1
-	peace_cost_factor = 100
+	peace_cost_factor = 85
```

Comment:
What's the reasoning here? AI can't actually enforce the cb when it's at 100?

Reply:
Yeah